### ISSUE TITLE
Notarise macOS App

### ISSUE THREAD

**Issue by raymondjxu** [BetterDiscord/Installer#126]
macOS complains that BetterDiscord is malware because Apple hasn't checked it (in typical Apple fashion.

I'd like for a team member to sign it and send it to Apple for notarisation (it's free for macOS) with each update so macOS stops complaining that it contains malware.

Alternative: have users override the prompt saying that it's malware (which at least IMO doesn't build as much user trust).

**Comment by tropicaaal** [comment]
Apple isn't especially clear on this from their documentation, but with a bit of reading, notarizing requires a Developer ID Certificate which is generated through XCode, however to do this you need to be in their developer program which is $99 a year.

**Comment by raymondjxu** [comment]
@Tropix126 No. Having a developer account and having a developer membership are different. Unless something has changed in the past month, it definitely is free, and I know this for sure *because I've signed apps before*.

**Comment by tropicaaal** [comment]
Ah got it, reopening.

**Comment by tropicaaal** [comment]
I'll talk with @rauenzi about this at some point.

**Comment by zerebos** [comment]
I tried for several hours to do this for free, and ran into the same issue as many people. It requires the $99 developer program https://stackoverflow.com/questions/60745167/the-username-is-not-a-member-of-the-provider

**Comment by raymondjxu** [comment]
I’ve done it before. It’s definitely possible. The StackOverflow answer is wrong (also, I've found that often the people answering their own questions don't give full and accurate answers).
